pkg/http/server/server.go

```diff
@@ -13,6 +14,17 @@ func New(cfg *Config) *http.Server {
 	}
 }
 
+func NewTLS(cfg *Config) *http.Server {
```

Comment:
I cannot find any usage of this method. I supposed if the use enables should specify whether or not he want to enable it depending on the values over cmd flags

Reply:
This bit has been missing too